Essentials/src/main/resources/plugin.yml

```
@@ -525,8 +525,8 @@ commands:
     usage: /<command> [player]
     aliases: [etpacancel]
   tpdeny:
-    description: Reject a teleport request.
-    usage: /<command>
+    description: Rejects most recent teleport request or one from a specific player or all tpa requests.
```

Same comment as with tpaccept.
```suggestion
    description: Rejects teleport requests.
```